core/src/transaction.rs

```diff
@@ -61,6 +106,17 @@ impl CellOutPoint {
     pub const fn serialized_size() -> usize {
         H256::size_of() + mem::size_of::<u32>()
     }
+
+    pub fn cell_key(&self) -> CellKey {
+        CellKey::calculate(&self.tx_hash, self.index)
+    }
+
+    pub fn to_ref(&self) -> CellOutPointRef {
```

Comment:
`as_ref`?

Reply:
`as_ref()` is used by `AsRef<T>`, and `as_ref()` will return a reference (`&T`), but `to_ref()` return `R` (not `&R`).